src/webaudio.js

```diff
@@ -257,6 +257,7 @@ WaveSurfer.WebAudio = {
         this.gainNode.disconnect();
         this.scriptNode.disconnect();
         this.analyser.disconnect();
+        this.ac.close();
```

Comment:
But that wouldn't fix use case #1 when an outside script is accessing WaveSurfer.WebAudio.audioContext  – Although I think that is probably a very highly exotic use case: using the audioContext, even after destroying the wavesurfer instance. – I think we could probably get away with doing what you propose without a large fallout.